src/components/Info/_info.scss

```diff
@@ -0,0 +1,47 @@
+.info-section__heading {
+  @include carbon--type-style('heading-01');
+}
+.info-card {
+  margin-top: $spacing-09;
+  display: flex;
+  flex-direction: column;
+
+  svg {
+    margin-top: $spacing-09;
+  }
+
+  // top border in only small breakpoints to prevent overrides
+  @include carbon--breakpoint-down(md) {
+    &:not(:nth-child(2)) {
+      border-top: 1px solid $ui-03;
+      padding-top: $spacing-09;
+    }
+  }
+
+  // left border in just the 2nd column items
+  @include carbon--breakpoint(md) {
+    &:nth-child(odd) {
+      border-left: 1px solid $ui-03;
+    }
+  }
+
+  // left border in all items
+  @include carbon--breakpoint(lg) {
+    margin-top: 0;
+    border-left: 1px solid $ui-03;
+
+    svg {
+      margin-top: $layout-06;
+    }
+  }
+}
+.info-card__body {
+  margin-top: $spacing-06;
+  flex-grow: 1; // fill space so icons are bottom aligned
+  @include type-style('body-long-01');
+
+  // prevent large line lengths between small and medium viewports
+  @include carbon--breakpoint-between(321px, md) {
+    max-width: 75%;
+  }
+}
```

Comment:
You're missing the `info-card__heading` class definition so your third word is not formatted correctly.

Reply:
Happy to correct it. Can you be a bit more specific, what you mean by 'third word'

Reply:
See this https://www.carbondesignsystem.com/tutorial/react-step-4#type

Reply:
Hi i checked the code ... info-card__heading is defined already and the style is applied so the third word is already in bold.

please check [my deployed app](http://carbon-tutorial-lalasmuathasim.eu-gb.mybluemix.net/#/). I'm sorry if missed anything from the instruction

Reply:
The class `info-card__heading` is not found in the file I am reviewing on GitHub. Have you pushed your updates to this PR?

Reply:
Thanks for the quick response. I will do new pull request from https://github.com/lalasmuathasim/carbon-tutorial/pull/new/react-step-4 and go through the the steps 4 again and push to the step 4. I believe that is the step required to resolve this. Correct me if I'm wrong .. Thanks

Reply:
I have updated the changes requested .. please review